linux_os/guide/system/permissions/mounting/kernel_module_fat_disabled/rule.yml

```diff
@@ -0,0 +1,33 @@
+documentation_complete: true
+
+prodtype: rhel6,rhel7,rhel8,fedora,rhv4
+
+title: 'Disable Mounting of FAT filesystems'
+
+description: |-
+    {{{ describe_module_disable(module="vfat") }}}
+    This effectively prevents usage of this uncommon filesystem.
+
+rationale: |-
+    Linux kernel modules which implement filesystems that are not needed by the
+    local system should be disabled.
+
+severity: low
+
+identifiers:
+    cce@rhel6: cce-82167-8
+    cce@rhel7: cce-82169-4
+    cce@rhel8: cce-82170-2
```

Comment:
@70k10 please remove the `cce-` from each of the CCEs

Reply:
doh, cut/paste fail. change made